ui/cypress/integration/tests.js

```diff
@@ -4,5 +4,17 @@ describe("Basic tests", () => {
 
     cy.contains("underlay_name").click();
     cy.contains("Datasets");
+
+    cy.get("button[id=insert-cohort]").click();
+    cy.get("button:Contains(Create)").click();
+    cy.get("a[id^=edit-cohort]").click();
+    cy.get("button:Contains(Add Criteria)").click();
+    cy.get("li:Contains(Conditions)").click();
+    cy.get("button[role=checkbox]").click();
+    cy.get("a[aria-label=back]").click();
+    cy.get("a[aria-label=back]").click();
+    cy.get("input[name='New Cohort']").click();
+    cy.get("input[name=Demographics]").click();
+    cy.contains("SELECT *");
```

Comment:
I haven't used cypress before. Am I correct in understanding this just asserts that the text "SELECT *" appears anywhere on the page?

Should the selector be more targeted?